ompi/proc/proc.h

```diff
@@ -74,6 +75,11 @@ struct ompi_proc_t {
     /* endpoint data */
     void *proc_endpoints[OMPI_PROC_ENDPOINT_TAG_MAX];
 
+#if OPAL_ENABLE_FT_MPI
+    /** Is the process active? */
+    bool                            proc_active;
```

Comment:
OpenSHMEM does use the padding, so it needs to remain available for that usage. We however definitely need to keep the field aligned for OpenSHMEM.

Reply:
I think I would just always leave this boolean in the proc structure and avoid any silly ABI questions.  That said, I would improve the comment so that it in some way references FT in the definition of "active".